nixos/modules/hardware/wooting.nix

```
@@ -1,12 +1,51 @@
 { config, lib, pkgs, ... }:
 
 with lib;
+let
+  cfg = config.hardware.wooting;
+  xinput-rules = pkgs.writeTextFile {
+    name = "71-wooting-xinput.rules";
+    text = ''
+      # Wooting One
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="3eb/ff01/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@03eb:ff01"
+
+      # Wooting Two
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="3eb/ff02/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@03eb:ff02"
+
+      # Wooting Two Lekker Edition
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="31e3/1210/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@31e3:1210"
+
+      # Wooting Two HE
+      SUBSYSTEM=="usb", ENV{PRODUCT}=="31e3/1220/*", ENV{INTERFACE}=="255/93/1", TAG+="systemd", ENV{SYSTEMD_WANTS}="wooting-xinput@31e3:1220"
+    '';
+  };
+in
 {
-  options.hardware.wooting.enable =
-    mkEnableOption "Enable support for Wooting keyboards";
+  options.hardware.wooting = {
+    enable = mkEnableOption "support for Wooting keyboards";
+    xinput.enable = mkEnableOption "xinput support for Wooting keyboards";
```

We should probably mention that this is an `xboxdrv`-based solution. In theory a different xinput driver (like `xone`?) should also work.

```suggestion
    xinput.enable = mkEnableOption "xinput support for Wooting keyboards (via xboxdrv)";
```